main/REPL/timerServices.md

```diff
@@ -0,0 +1,119 @@
+# Timer Service Objects
+
+`home` has two timer service objects: 
+- `localTimerService` is from the local vat.
+- `chainTimerService` is from the chain (if present). 
+
+In general, you should use the chain timer for contracts. But it's more efficient to 
+use the local timer for objects that don't need consensus or consistency. 
+For example, people might want it for testing. 
+
+Both timer services let you:
+- Get the start time of the current block. **tyg todo: Is this the case for local?**
+- Schedule `wake()` calls.
+- Remove scheduled `wake()` calls.
+- Create a repeater for scheduling of events at regular intervals. 
+
+With respect to using them, the only difference between the two timer
+services is whether you preface their identical command set with
+`localTimerService` or `chainTimerService`. However, note that
+the local and chain versions of a method may use different time bases.
+
+Before looking at the timer service methods, we need to cover two related objects,
+the `TimerWaker` and the `TimerRepeater`. 
+
+A `TimerWaker` has one method, `wake`, which takes a `Timestamp` argument and returns `void`. 
+The passed `timestamp` is the time when the call to `wake()` is scheduled to occur.
+
+A `TimerRepeater` has two methods, `schedule()` and `disable`:
+- `schedule(waker)`
+  - `waker` `{ `TimerWaker` }`
+  - Returns `void`             **tyg: The source ChrisH passed along was @property {(waker: TimerWaker) => void} Should it be a Timestamp instead of void?**
```

Comment:
yes, `schedule(waker)` returns a Timestamp indicating the time the waker is next scheduled to be called.